CMakeLists.txt

```diff
@@ -0,0 +1,740 @@
+# Copyright 2017 The LEVELDB Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+cmake_minimum_required(VERSION 3.1)
+project(leveldb VERSION 0.1.0 LANGUAGES C CXX)
```

Comment:
`C CXX` is the default, no need to specify it explicitly. From [documentation](https://cmake.org/cmake/help/latest/command/project.html):
```
By default C and CXX are enabled if no language options are given.
```